packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.html

```diff
@@ -6,7 +6,7 @@
     <div class="slds-size_1-of-1">
       <c-quantic-card-container title={label}>
         <lightning-button-icon
-          class={actionButtonCssClasses}
+          class="actionButtonCssClasses"
```

Comment:
`{actionButtonCssClasses}`

Reply:
i dont understand where this came from...